package.json

```diff
@@ -33,7 +33,7 @@
     "url": "https://github.com/iTwin/projects-client"
   },
   "dependencies": {
-    "@bentley/itwin-client": "3.0.0-dev.138"
+    "axios": "^0.25.0"
```

Comment:
Is there any risk in relying on a library in version 0? And yes I see the irony that this package is version 0 as well.

Reply:
yea is def a risk, but axios is a very popular lib and should be ok for this

Reply:
I hadn't heard of it before, but maybe I should look into it because it seems very useful